src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/InputUnusedImportFromJavaLang.java

```diff
@@ -0,0 +1,44 @@
+package com.puppycrawl.tools.checkstyle.checks.imports;
+
+import java.lang.String; // VIOLATION
+import java.lang.Math; // VIOLATION
+import java.lang.Class; // VIOLATION
+import java.lang.Exception; // VIOLATION
+import java.lang.Runnable; // VIOLATION
+import java.lang.RuntimeException; // VIOLATION
+import java.lang.ProcessBuilder; // VIOLATION
+import java.lang.Double; // VIOLATION
+import java.lang.Integer; // VIOLATION
+import java.lang.Float; // VIOLATION
+import java.lang.Short; // VIOLATION
+
+import java.lang.annotation.Annotation; // OK
+import java.lang.reflect.Modifier; // OK
+import java.lang.reflect.Field; // OK
+
+import java.lang.*; // OK
```

Comment:
I think it is OK to skip on demand import, there are numbers of nuances when it come to conflicting names of classes in different psckages , we skip them for others imports. So let's keep behavior the same.